Doc/reference/import.rst

```diff
@@ -922,6 +922,53 @@ it is sufficient to raise :exc:`ModuleNotFoundError` directly from
 while raising an exception terminates it immediately.
 
 
+Package Relative Imports
+========================
+
+Relative imports use leading dots. A single leading dot indicates a relative
+import, starting with the current package. Two or more leading dots indicate a
+relative import to the parent(s) of the current package, one level per dot
+after the first. For example, given the following package layout::
+
+    package/
+        __init__.py
+        subpackage1/
+            __init__.py
+            moduleX.py
+            moduleY.py
+        subpackage2/
+            __init__.py
+            moduleZ.py
+        moduleA.py
+
+Given that the current file is either ``moduleX.py`` or ``subpackage1/__init__.py``,
```

Comment:
I would say "If the current file is" rather than "Given that the current file is". When I read "given that..." I take it as a fact (or a consequence of earlier assumptions), rather than introducing a new assumption.

Alternatively, this sentence can be reworded as:

"In either `subpackage1/moduleX.py` or `subpackage1/__init__.py`, the following relative imports are valid:"